android/src/main/java/com/tailscale/ipn/ui/localapi/LocalAPIClient.kt

```diff
@@ -46,73 +46,57 @@ class LocalApiClient {
     // body: The body of the request.
     // cookie: A unique identifier for this request.  This is used map responses to
     //         the corresponding request.  Cookies must be unique for each request.
-    external fun doRequest(request: String, method: String, body: String, cookie: String)
+    private external fun doRequest(request: String, method: String, body: ByteArray?, cookie: String)
```

Comment:
Using a ByteArray for request and response bodies should be more efficient than using strings. Eventually, I might even want to change the API to use streams, if I can figure out how to do that across the JNI boundary, but not worth chewing off right now.

Reply:
Also, some things are not well suited to strings, for example Taildrop file PUTs.